src/set.ts

```diff
@@ -0,0 +1,145 @@
+const gameResolution = { width: 400, height: 300 };
```

Comment:
this is all throwaway code at this point! just trying to render _something_ to the screen as a start. includes some experiments with the dashed fill and attempting (unsuccessfully) to render an S shape.